tests/functional/src/test_document.py

```diff
@@ -1,5 +1,25 @@
 class TestDocument:
-    def test_document(self, firefox_client):
+    def test_ps(self, firefox_client):
         response = firefox_client.navigation.navigate('https://pychik.github.io/lightning_py/pages/1')\
-                                            .document.get_page_source()
+                                            .document.get_page_source
         assert response.startswith("<html")
+
+    def test_es(self, firefox_client):
+        res = firefox_client.document.execute_script("return arguments[0] + arguments[1];", 1, 2)
+        assert res == 3
+        res = firefox_client.document.execute_script("return arguments[0] + arguments[1];", "1", "2")
+        assert res == "12"
+
+    def test_eas(self, firefox_client):
```

Comment:
Rename to `execute_async_script`.

Reply:
solved in b3eabe6a0fda5fc9fd50d8fa8da77df258cc7342